webpack.mix.js

```diff
@@ -56,3 +72,7 @@ mix.webpackConfig({
     },
   },
 });
+
+if (mix.inProduction()) {
```

Comment:
I don't see the advantage of only versioning in production. Old cached js files can cause plenty of confusion in dev environments.